.eslintrc.js

```diff
@@ -1576,20 +1625,5 @@ module.exports = {
         '@typescript-eslint/prefer-ts-expect-error': 'error',
       },
     },
-    {
-      files: [
-        '**/public/**/*.{js,mjs,ts,tsx}',
-        '**/common/**/*.{js,mjs,ts,tsx}',
-        'packages/**/*.{js,mjs,ts,tsx}',
-      ],
-      rules: {
-        'no-restricted-imports': [
-          'error',
-          {
-            patterns: ['lodash/*', '!lodash/fp', 'rxjs/internal-compatibility'],
-          },
-        ],
-      },
-    },
```

Comment:
I removed this rule because it prevented the other `no-restricted-imports` rule above from firing. Even combining them together did not fix the problem. This is apparently desired behavior: https://github.com/eslint/eslint/issues/14220

I opted to remove this rule in favor of the definition above because the other definitions are security-related, which is [subjectively] much more important.